web/src/components/apps/AppVersionHistory.jsx

```diff
@@ -1145,6 +1145,12 @@ class AppVersionHistory extends Component {
       checkingUpdateTextShort = checkingUpdateTextShort.slice(0, 30) + "...";
     }
 
+    const renderKotsUpgradeStatus = this.state.kotsUpdateStatus && !this.state.kotsUpdateMessage;
```

Comment:
why not show them together? just curious

Reply:
i don't have a snapshot, but it looks weird, confusing, and redundant